src/main/java/seedu/module/logic/commands/FindCommand.java

```diff
@@ -28,9 +29,10 @@ public FindCommand(NameContainsKeywordsPredicate predicate) {
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.updateFilteredModuleList(predicate);
+        model.updateFilteredArchivedModuleList(predicate);
+        model.changeDisplayedList("ArchivedList");
```

Comment:
Same as above, use of magic word.

Reply:
Split changeDisplayedList to displayTrackedList and displayArchivedList.